oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/FullGCStatsCollectorImpl.java

```diff
@@ -37,7 +37,8 @@
  */
 class FullGCStatsCollectorImpl implements FullGCStatsCollector {
 
-    static final String FULL_GC = "FullGC";
+    static final String FULL_GC_PUSH_METRICS_PREFIX = "oak_FullGC";
+    static String FULL_GC = "FullGC";
```

Comment:
Do we really need to remove the `final` here?

```suggestion
    static final String FULL_GC = "FullGC";
```

Reply:
Fixed, thank you for the suggestion